pages/wishlistPage/wishlistPage.html

```diff
@@ -0,0 +1,418 @@
+                            <p
+                                class="dui-card__secondary-text dui-util-txt-sm dui-util-spc-pad-0_8rem-xs dui-util-fw-sbld">
+                                50%
+                                OFF</p>
+                        </div>
+
+                        <div class="dui-card__actions dui-util-spc-pad-1_6rem-s">
+                            <div class="dui-card__buttons dui-card__buttons--hr-prod">
+                                <button
+                                    class="dui-btn dui-btn--hr-prod dui-btn--primary dui-util-txt-sm dui-util-spc-pad-0_8rem-xs reset-button-inherit-parent">Move
+                                    to Cart</button>
+                                <button
+                                    class="dui-btn dui-btn--hr-prod dui-btn--secondary dui-util-txt-sm dui-util-spc-pad-0_8rem-xs reset-button-inherit-parent">Remove
+                                    from Wishlist</button>
+                            </div>
+
+                        </div>
+                        <!-- Button Component Starts -- Icon -->
+                        <button
+                            class="main-content__wish-list-card-btn dui-btn dui-util-bdr-radi-999px-mx reset-button-inherit-parent">
+                            <svg xmlns="http://www.w3.org/2000/svg"
+                                class="dui-util-spc-pad-0_8rem-xs icon icon-tabler icon-tabler-heart" width="56"
+                                height="56" viewBox="0 0 24 24" stroke-width="1.5" fill="#F34E4E" fill="none"
+                                stroke-linecap="round" stroke-linejoin="round">
+                                <path stroke="none" d="M0 0h24v24H0z" fill="none" />
+                                <path
+                                    d="M19.5 13.572l-7.5 7.428l-7.5 -7.428m0 0a5 5 0 1 1 7.5 -6.566a5 5 0 1 1 7.5 6.572" />
+                            </svg>
+                        </button>
+                        <!-- Button Component Ends -- Icon -->
+                    </div>
+                    <!-- Card Component Ends -- Product Horizontal -->
+
+                    <!-- Card Component Starts -- Product Horizontal -->
+                    <div
+                        class="main-content__wish-list-card dui-card dui-card--hr-prod dui-util-bdr-radi-5px-s dui-util-gry-shdw dui-util-pos-rel">
+
+                        <!-- Badge Component Starts -- with Text -->
+                        <div class="dui-badge">
+                            <div class="dui-util-txt-sm dui-util-fw-reg">
+                                SALE
+                            </div>
+                        </div>
+                        <!-- Badge Component Ends -- with Text -->
+
+                        <img class="dui-card__img dui-card__img--hr-prod"
+                            src="/assets/potted-plant-gda1ef23cd_640.png" />
```

Comment:
pngs files are the heaviest of all image file types

Reply:
Right now these are just placeholders for the UI and not actual images.

will be using **srcset** while optimizing for performance.

Reply:
Okay nice!